docs/index.md

```
@@ -1,18 +1,22 @@
+  - navigation
+---
 
-## Overview
+# Overview
 
-### What is MVI
+## What is MVI
 
 MVI stands for Model-View-Intent. It is an architectural pattern that utilizes unidirectional data flow. The data circulates between `Model` and `View` only in one direction - from `Model` to `View` and from `View` to `Model`.
 
 <img src="media/mvi.jpg" width="256">
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)